src/index.ts

```diff
@@ -11,7 +11,7 @@ interface DebugLogger extends DebugLoggerFunction {
  * Enum of metrics types
  * @see https://github.com/statsd/statsd/blob/master/docs/metric_types.md
  */
-enum Types {
```

Comment:
Here for extendibility, we have
```ts
// Public methods
    getSupportedTypes(): typeof Types {
        return Types;
    }
```
You don't need to export it.

Reply:
I don't like the fact that I have to instantiate the class just to get the types, maybe that should be a static method

Reply:
okay

Reply:
Thank you for your suggestion, but after further review of the proposed change, I still believe that exporting the types is the best course of action in this scenario. It allows for flexibility in usage, as projects using plain JavaScript can utilize the getSupportedTypes method, while TypeScript users can benefit from using the Enum directly!

I have also decided to abandon my previous proposal. Turning this into a static method would introduce a breaking change without a clear benefit!